package/utils/policycoreutils/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=policycoreutils
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/20200710
+PKG_HASH:=c889f62ee80f8b6a369469a9b8af51f5b797975aeaa291f5c5960cc12eed1934
+PKG_INSTALL:=1
+HOST_BUILD_DEPENDS:=libsemanage/host
+PKG_BUILD_DEPENDS:=BUSYBOX_CONFIG_PAM:libpam
+
+PKG_MAINTAINER:=Thomas Petazzoni <thomas.petazzoni@bootlin.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/policycoreutils
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:= +libsemanage +libcap-ng +BUSYBOX_CONFIG_PAM:libpam
+  TITLE:=SELinux policy utilities
+  URL:=http://selinuxproject.org/page/Main_Page
+  ALTERNATIVES:=\
+    300:/sbin/restorecon:policycoreutils-restorecon \
+    300:/sbin/setfiles:policycoreutils-setfiles \
+    300:/usr/sbin/load_policy:policycoreutils-load_policy \
+    300:/usr/sbin/sestatus:policycoreutils-sestatus \
+    300:/usr/sbin/setsebool:policycoreutils-setsebool
+endef
+
+define Package/policycoreutils/description
+	Policycoreutils is a collection of policy utilities
+	(originally the "core" set of utilities needed to use
+	SELinux, although it has grown a bit over time), which have
+	different dependencies.  sestatus, secon, run_init, and
+	newrole only use libselinux. load_policy and setfiles only
+	use libselinux and libsepol. semodule and semanage use
+	libsemanage (and thus bring in dependencies on libsepol and
+	libselinux as well). setsebool uses libselinux to make
+	non-persistent boolean changes (via the kernel interface)
+	and uses libsemanage to make persistent boolean changes.
+endef
+
+include $(INCLUDE_DIR)/host-build.mk
+
+HOST_MAKE_FLAGS += \
+	PAMH=$(CONFIG_BUSYBOX_CONFIG_PAM) \
+	PREFIX=$(STAGING_DIR_HOSTPKG) \
+	SBINDIR=$(STAGING_DIR_HOSTPKG)/sbin \
+	ETCDIR=$(STAGING_DIR_HOSTPKG)/etc
+
+MAKE_FLAGS += \
+	PAMH=$(CONFIG_BUSYBOX_CONFIG_PAM)
+
+define Package/policycoreutils/install
```

Comment:
It'd be nice to break this up into individual packages, afaik also the optional libpam dependency then would only affect the `newrole` package (and then we could even have variants: `newrole` and `newrole-libpam`). See package [`util-linux`](https://github.com/openwrt/openwrt/blob/master/package/utils/util-linux/Makefile) for a good example.

However, I wouldn't mind to do that after the initial merge of that PR which is waiting too long already.

Reply:
I would greatly appreciate allowing this to happen post-merge.